plex_trakt_sync/commands/plex_login.py

```diff
@@ -171,7 +171,14 @@ def plex_login(username, password):
     CONFIG["PLEX_USERNAME"] = user
     CONFIG["PLEX_TOKEN"] = token
     CONFIG["PLEX_BASEURL"] = plex._baseurl
-    CONFIG["PLEX_FALLBACKURL"] = "http://localhost:32400"
+    if environ.get("PTS_IN_DOCKER"):
+        try:
+            host_ip = check_output("ip -4 route show default | awk '{ print $3 }'", shell=True).decode().rstrip()
```

Comment:
perhaps we should try host.docker.internal name first. it exists in docker for mac and docker for windows

- https://docs.docker.com/desktop/mac/networking/

```
➔ docker-compose run --entrypoint 'sh -c'  plex-trakt-sync  'getent hosts host.docker.internal'
Creating plextraktsync_plex-trakt-sync_run ... done
192.168.65.2      host.docker.internal  host.docker.internal
```

```
➔ docker-compose run --entrypoint 'python -c'  plex-trakt-sync  'import socket; print(socket.gethostbyname("host.docker.internal"));'
Creating plextraktsync_plex-trakt-sync_run ... done
192.168.65.2
```